shared/components/Animation/__tests__/Reveal.spec.jsx

```diff
@@ -24,3 +24,25 @@ describe('Reveal', () => {
     expect(reveal).toMatchSnapshot()
   })
 })
+
+describe('accessibility', () => {
```

Comment:
Small picky thing... our test convention is to next all tests within a top level `describe`. Nested `describe`s are fine. 

So this test file may look like:

```js
describe('Reveal', () => {
  it('test 1', () => {
  })

  it('test 2', () => {
  })

  describe('nested behaviours only if necessary because there are a lot of tests', () => {
    it('test 3', () => {
    })
  })
})
```

But in this case, there are only a few tests so the nested `describe` is probably overkill.